### PR TITLE
build: release Lotus Node and Miner v1.32.0-rc1

### PR DESCRIPTION
**Pull request by rjan90**
## Related Issues
https://github.com/filecoin-project/lotus/issues/12750

## Proposed Changes
Release v1.32.0-rc1

## Checklist

Before you mark the PR ready for review, please make sure that:

- [x] Commits have a clear commit message.
- [x] PR title conforms with [contribution conventions](https://github.com/filecoin-project/lotus/blob/master/CONTRIBUTING.md#pr-title-conventions)
- [x] Update CHANGELOG.md or signal that this change does not need it per [contribution conventions](https://github.com/filecoin-project/lotus/blob/master/CONTRIBUTING.md#changelog-management)
- [ ] New features have usage guidelines and / or documentation updates in
  - [ ] [Lotus Documentation](https://lotus.filecoin.io)
  - [ ] [Discussion Tutorials](https://github.com/filecoin-project/lotus/discussions/categories/tutorials)
- [ ] Tests exist for new functionality or change in behavior
- [ ] CI is green

### CODE REVIEW

**Comment by rjan90** [comment]
Release drafts are here:
- https://github.com/filecoin-project/lotus/releases/tag/untagged-8ed42d5d6cc091122158
- https://github.com/filecoin-project/lotus/releases/tag/untagged-eb20e821bd1a088955d6

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,13 +1,51 @@
 # Lotus changelog
 
-# UNRELEASED
+# Node and Miner v1.32.0-rc1 / 2024-12-05
+
+This is the first release candidate of the upcoming MANDATORY Lotus v1.32.0 release, which will deliver the Filecoin network version 25, codenamed “Teep” 🦵.
+
+This release candidate sets the upgrade epoch for the calibration network to **Epoch 2235454 -  2024-12-16T23:00:00Z**. This should be the equivalent to:
```
Comment (rvagg) [inline CHANGELOG.md:7]:
```suggestion
This release candidate sets the upgrade epoch for the calibration network to **Epoch 2235454 - 2024-12-16T23:00:00Z**. This should be the equivalent to:
```

sorry (not sorry)

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,13 +1,51 @@
 # Lotus changelog
 
-# UNRELEASED
+# Node and Miner v1.32.0-rc1 / 2024-12-05
+
+This is the first release candidate of the upcoming MANDATORY Lotus v1.32.0 release, which will deliver the Filecoin network version 25, codenamed “Teep” 🦵.
+
+This release candidate sets the upgrade epoch for the calibration network to **Epoch 2235454 - 2024-12-16T23:00:00Z**. This should be the equivalent to:
+
+- 15:00 in Los-Angeles
```
Comment (rvagg) [inline CHANGELOG.md:9]:
```suggestion
- 15:00 in Los Angeles
```

Comment (rvagg) [reply CHANGELOG.md:9]: confirmed these times w/ https://www.worldtimebuddy.com/?qm=1&lid=100,2174003,2147714,1816670,2643743,5368361&h=100&date=2024-12-16&sln=23-24&hf=1

👍

**Review by rvagg — APPROVED** [review]

**Pull request merged**